### PR TITLE
V12 Update and Sharding

### LINKED ISSUE #769: Move message command
I've made a personal command for my guild where the staff can move a message from channel to another, we use it for message that shouldn't be on *x* channel because the content of *x* message is for *y* channel.

So i think that it would ne a nice moderation command on switchblade where the user need MANAGE_MESSAGES permission to use that command and the usage would be something like: `prefix.move <messageLink> <channelID>`, 
literal: `s!move https://discordapp.com/channels/445203868624748555/514534829744848906/556942345703129109 445631443490832394`

Message Link being the same argument that i've used here https://github.com/SwitchbladeBot/switchblade/issues/760#issuecomment-471322538

and channelID could be a channel mention or the channelID, this command would have rules like: you can't move a message from another guild because that would be bs, you can't move message from members that have a role where the position is the same or higher than your highest role & guild owner and you can't move messages from a nsfw channel to a not nsfw channel. (Nicely noted beescuit.)

### PR DESCRIPTION
**Pull request by Doges**
**switchblade now:**

This closes #1110, Closes #769, and Closes #1095

### CODE REVIEW

File: src/utils/PermissionUtils.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ module.exports = class PermissionUtils {
...
 
   static isManager (client, user) {
-    const botGuild = client.guilds.get(process.env.BOT_GUILD)
-    const managerRole = botGuild && botGuild.roles.get(process.env.MANAGER_ROLE)
-    const isManager = (managerRole && managerRole.members.has(user.id)) || this.isDeveloper(client, user)
+    const botGuild = client.guilds.cache.get(process.env.BOT_GUILD)
+    const managerRole = botGuild && botGuild.roles.cache.get(process.env.MANAGER_ROLE)
+    const isManager = (managerRole && managerRole.members.cache.has(user.id)) || this.isDeveloper(client, user)
     return isManager
   }
```
Comment (pedrofracassi) [inline src/utils/PermissionUtils.js]: won't work cause the guild might not be on the same shard

Comment (Doges) [reply src/utils/PermissionUtils.js]: yea, thats expected

Comment (moruzerinho6) [reply src/utils/PermissionUtils.js]: O boy, I know nothing about sharding.

Comment (Doges) [reply src/utils/PermissionUtils.js]: that will also happen with all profile, leaderboard, userinfo and guildinfo

**Comment by metehus** [comment]
damn

**Comment by Doges** [comment]
Ready for review, that can break the entire project, please pay attention.

**Pull request merged**